config/schedule.rb

```diff
@@ -45,17 +45,12 @@
   ThreeScale::Jobs::BILLING.each { |task| instance_exec(task, &job_proc) }
 end
 
+every 1.day, :at => '6:00 am', roles: [:sphinx_server] do
+  ThreeScale::Jobs::SPHINX_INDEX_ALL.each { |task| instance_exec(task, &job_proc) }
+end
+
 ThreeScale::Jobs::CUSTOM.each_pair do |interval, task|
   every interval, roles: [:cron] do
     instance_exec(task, &job_proc)
   end
 end
-
-every 1.day, :at => '6:00 am', roles: [:sphinx_server] do
-  ThreeScale::Jobs::SPHINX_INDEX.each { |task| instance_exec(task, &job_proc) }
-end
-
-# every hour would put at minute 0 which is quite busy with tasks already
-every '42 * * * *', roles: [:sphinx_server] do
-  ThreeScale::Jobs::SPHINX_DELTA.each { |task| instance_exec(task, &job_proc) }
-end
```

Comment:
Need to remove from https://github.com/3scale/porta/blob/1ef7f18809ec074bebe396ded6436a2f4c1da64f/openshift/system/config/sidekiq_schedule.yml#L39-L51 too?